pkg/cluster/config.go

```diff
@@ -107,23 +109,28 @@ func NewConfig(restConfig *rest.Config) (Config, error) {
 	}, nil
 }
 
-// validateK8sVersion checks for valid k8s version in the cluster. It returns an error for all versions not equal
-// to supported major version. This is being done this way, and not by directly getting the cluster version, as OpenShift CI
-// returns version in the format 0.0.x and not the actual version attached to its clusters.
+// validateK8sVersion checks for valid k8s version in the cluster. It returns an error for all versions that are not in
+// range of given base version(x.y.z) and x.y+1.z version. This is being done this way, and not by directly getting
+// the cluster version, as OpenShift CI returns version in the format 0.0.x and not the actual version attached to its
+// clusters.
```

Comment:
> OpenShift CI returns version in the format 0.0.x and not the actual version attached to its clusters.

This is no longer true. I suggest we remove this from the comment. On that note: should we start using OpenShift versions now?

Reply:
I will remove the comment here. 

As discussed in the standup:
The consensus was to start using `OpenShift versions` since we are releasing our operator for particular OpenShift versions. There is a [story](https://issues.redhat.com/browse/WINC-518)  regarding this, so the changes can be made when we decide to undertake it.